NomadAudio/test/AudioEngineSoakTest.cpp

```
@@ -0,0 +1,371 @@
+        }
+    }
+
+    running.store(false, std::memory_order_release);
+    stress.join();
+
+    const auto endWall = std::chrono::steady_clock::now();
+    const double wallSec = std::chrono::duration_cast<std::chrono::duration<double>>(endWall - startWall).count();
+    const double avgNs = (blocks > 0) ? static_cast<double>(sumCallbackNs / static_cast<long double>(blocks)) : 0.0;
+
+    // Drift: engine increments global sample pos while transport is playing.
+    const uint64_t expectedSamples = blocks * static_cast<uint64_t>(opt.bufferFrames);
```

Clang (14.0.6)

[warning] variable 'expectedSamples' is not initialized

(cppcoreguidelines-init-variables)